LoopKitUI/CGMManagerUI.swift

```diff
@@ -9,15 +9,20 @@ import LoopKit
 import HealthKit
 
 
-public protocol CGMManagerUI: CGMManager {
+public protocol DeviceManagerUI {
+    // An image representing the device configuration
+    static var smallImage: UIImage? { get }
+    static var name: String { get }
```

Comment:
A static variable `localizedTitle` already exists as a requirement of `DeviceManager`. What need does `name` fulfill?

Reply:
TBH I wasn't sure if `localizedTitle` fulfills the same need... maybe @ps2 can weigh in. That said, `DeviceManagerUI` is specifically meant only for the device's UI representation, here:
<img width="531" alt="Screen Shot 2020-06-30 at 2 49 18 PM" src="https://user-images.githubusercontent.com/308993/86180405-ed021a00-bae0-11ea-8008-3f943ad2a0e0.png">
If that's what `localizedTitle` is for, then maybe it belongs here instead...?